2023-12-18 17:08:48,329 INFO    MainThread:3876 [wandb_setup.py:_flush():76] Current SDK version is 0.16.1
2023-12-18 17:08:48,329 INFO    MainThread:3876 [wandb_setup.py:_flush():76] Configure stats pid to 3876
2023-12-18 17:08:48,329 INFO    MainThread:3876 [wandb_setup.py:_flush():76] Loading settings from C:\Users\rs1044\.config\wandb\settings
2023-12-18 17:08:48,329 INFO    MainThread:3876 [wandb_setup.py:_flush():76] Loading settings from c:\Users\rs1044\Documents\GitHub\Mixture-of-Experts-based-Federated-Learning-for-Energy-Forecasting\src\time_series\Federatedlearning\TS_FL_Grossload_Forecasting_Benchmark\wandb\settings
2023-12-18 17:08:48,329 INFO    MainThread:3876 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'silent': 'true', 'console': 'off'}
2023-12-18 17:08:48,339 INFO    MainThread:3876 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-12-18 17:08:48,339 INFO    MainThread:3876 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-12-18 17:08:48,339 ERROR   MainThread:3876 [wandb_setup.py:_flush():76] [34m[1mwandb[0m: [32m[41mERROR[0m Control-C detected -- Run data was not synced
2023-12-18 17:08:48,339 INFO    MainThread:3876 [wandb_init.py:_log_setup():524] Logging user logs to c:\Users\rs1044\Documents\GitHub\Mixture-of-Experts-based-Federated-Learning-for-Energy-Forecasting\src\time_series\Federatedlearning\TS_FL_Grossload_Forecasting_Benchmark\wandb\run-20231218_170848-ocojk7t7\logs\debug.log
2023-12-18 17:08:48,340 INFO    MainThread:3876 [wandb_init.py:_log_setup():525] Logging internal logs to c:\Users\rs1044\Documents\GitHub\Mixture-of-Experts-based-Federated-Learning-for-Energy-Forecasting\src\time_series\Federatedlearning\TS_FL_Grossload_Forecasting_Benchmark\wandb\run-20231218_170848-ocojk7t7\logs\debug-internal.log
2023-12-18 17:08:48,340 INFO    MainThread:3876 [wandb_init.py:_jupyter_setup():470] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000012227F2CE50>
2023-12-18 17:08:48,340 INFO    MainThread:3876 [wandb_init.py:init():564] calling init triggers
2023-12-18 17:08:48,340 INFO    MainThread:3876 [wandb_init.py:init():571] wandb.init called with sweep_config: {}
config: {'max_epochs': 2, 'batch_size': 16, 'optimizer': 'Adam', 'learning_rate': 0.001, 'architecture': 'global_bilstm'}
2023-12-18 17:08:48,340 INFO    MainThread:3876 [wandb_init.py:init():614] starting backend
2023-12-18 17:08:48,340 INFO    MainThread:3876 [wandb_init.py:init():618] setting up manager
2023-12-18 17:08:48,341 INFO    MainThread:3876 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2023-12-18 17:08:48,342 INFO    MainThread:3876 [wandb_init.py:init():624] backend started and connected
2023-12-18 17:08:48,345 INFO    MainThread:3876 [wandb_run.py:_label_probe_notebook():1294] probe notebook
2023-12-18 17:08:48,345 INFO    MainThread:3876 [wandb_run.py:_label_probe_notebook():1304] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-12-18 17:08:48,345 INFO    MainThread:3876 [wandb_init.py:init():716] updated telemetry
2023-12-18 17:08:48,395 INFO    MainThread:3876 [wandb_init.py:init():749] communicating run to backend with 90.0 second timeout
